src/configs/data/mvtec.yaml

```diff
@@ -0,0 +1,18 @@
+class_path: anomalib.data.MVTec
```

Comment:
Not in this PR, but I think it is time to rename this to MVTecAD. This is the original name of this dataset, and we have multiple MVTec datasets now